src/main/java/uk/gov/hmcts/cmc/claimstore/services/FormaliseResponseAcceptanceService.java

```diff
@@ -55,13 +56,22 @@ public void formalise(Claim claim, ResponseAcceptation responseAcceptation, Stri
                 formaliseSettlement(claim, responseAcceptation, authorisation);
                 break;
             case REFER_TO_JUDGE:
-                eventProducer.createInterlocutoryJudgmentEvent(claim);
+                createEventForReferToJudge(claim);
                 break;
             default:
                 throw new IllegalStateException("Invalid formaliseOption");
         }
     }
 
+    private void createEventForReferToJudge(Claim claim) {
```

Comment:
@rattur ignore those comments (I deleted anyway), I see we need here 👍

Reply:
I think we can delete this validator ValidResponseAcceptanceConstraintValidator